src/saturn_engine/worker/pipeline_message.py

```diff
@@ -16,6 +16,9 @@ class PipelineMessage:
     message: TopicMessage
     meta_args: dict[str, t.Any] = dataclasses.field(default_factory=dict)
 
+    def __post_init__(self) -> None:
```

Comment:
To avoid copying the message twice when it's going to get pickled, I would suggest to do this in `PipelineBootstrap::bootstrap_pipeline`

Reply:
Got it.

Where do you suggest writing the test, `test_jobs` or `test_services`?

Reply:
Hum, seems we don't have any test coverage for `BootstrapPipeline`, but that should be somewhat easy to add a single test. It's pretty much your current test with a `BootstrapPipeline.run_pipeline` instead of calling directly `msg.execute()`

Reply:
(I would suggest adding a new file `test_bootstrap.py`